### PR TITLE
WEB: Use own domain to HTTPS, outbound where redirected

### PR DESCRIPTION
**Pull request by Botje**
Tested locally with Docker.

### CODE REVIEW

File: include/config.inc.php [hunk 1]
```diff
@@ -13,18 +13,21 @@
 /* Time zone to use for news items etc. */
 date_default_timezone_set("UTC");
 
+$proto = (isset($_SERVER["HTTPS"]) && $_SERVER["HTTPS"]) == "on" ? "https": "http";
+
 /* Base URL to the website. */
 if ($_SERVER['SERVER_PORT'] == '80') {
-	$url = "http://{$_SERVER['SERVER_NAME']}" . dirname($_SERVER['PHP_SELF']);
+	$url = "{$proto}://{$_SERVER['SERVER_NAME']}" . dirname($_SERVER['PHP_SELF']);
```
Comment (klusark) [inline include/config.inc.php]: Don't actually need to include a proto. If it's blank it will just use whatever the page was loaded with.

Comment (Botje) [reply include/config.inc.php]: `baseurl` is derived from the `BASE_URL` constant defined in this section.
RSS feeds also use `baseurl`. I have no idea what `//residualvm.org` will resolve to, there.

To be honest I'd prefer to make the site HTTPS-only, like ScummVM.

Comment (aquadran) [reply include/config.inc.php]: does it solve it? https://test.residualvm.org

Comment (aquadran) [reply include/config.inc.php]: website already has redirections to https. but main page images seems are not redirected somehow

Comment (aquadran) [reply include/config.inc.php]: I didn't notice before port in url. remove it, fixed redirection

Comment (aquadran) [reply include/config.inc.php]: btw. you pushed your branch on this repo.

**Pull request merged**